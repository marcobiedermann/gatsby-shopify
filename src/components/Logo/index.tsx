import { Link } from 'gatsby';
import React, { ReactNode } from 'react';

export interface LogoProps {
  children: ReactNode;
}

function Logo(props: LogoProps) {
  return <Link to="/" {...props} />;
};

export default Logo;
