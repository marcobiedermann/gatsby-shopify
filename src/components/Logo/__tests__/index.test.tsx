import { render } from '@testing-library/react';
import React from 'react';
import Logo from '..';

describe('logo component', () => {
  it('should render correctly', () => {
    const { container } = render(<Logo>Logo</Logo>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
