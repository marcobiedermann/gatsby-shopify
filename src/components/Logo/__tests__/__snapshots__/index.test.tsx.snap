// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo component should render correctly 1`] = `
<a
  href="/"
>
  Logo
</a>
`;
